bin/install-dependencies.sh

```diff
@@ -18,6 +18,7 @@ pycsw_tag='2.4.0'
 pipopt='--exists-action=b'
 
 $pip install $pipopt -U numpy==1.16.4
+$pip install $pipopt -U setuptools_rust
```

Comment:
Worth adding in the commit message why this is needed?

Reply:
decided to remove it as no longer needed after pip upgrade